test/test_plots.py

```diff
@@ -33,3 +33,12 @@ def test_plot_2d_contour(self):
 
     def test_plot_T_c_inv_along_div(self):
         plot_T_c_inv_along_divertor(filenames=[self.filename, self.filename], filetypes="WEST")
+
+def test_plot_inv_with_uncertainty(self):
```

Comment:
The `self` argument here is used when inside a class (like the class `TestExtrudeStraightShape` above). This is why the CI is failing.

Here simply do:

```suggestion
def test_plot_inv_with_uncertainty():
```